gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml

```diff
@@ -0,0 +1,22 @@
+# This file contains env vars which will ONLY be used on Mozorg Demos on GCP.
+#
+# This file is public: do _not_ put secrets such as API keys into this file.
+# If you need to add a secret, please consult with SRE or MEAO backend to
+# learn how to do this.
+
+ALLOWED_HOSTS: .tekcopteg.org,.moz.works,.run.app
+CONTENT_CARDS_URL: https://www-dev.allizom.org/media/
+CSP_DEFAULT_SRC: "*.tekcopteg.com"
+CSP_EXTRA_FRAME_SRC: "*.mozaws.net,o1069899.sentry.io"
+CSP_REPORT_ENABLE: "True"
+DB_DOWNLOAD_IGNORE_GIT: "True"
+DEBUG: "False"
+DEV: "True"
+GTM_CONTAINER_ID: GTM-MW3R8V
```

Comment:
This is mozorg's GTM container ID, is it needed for the Pocket deploy also?

Reply:
Will check if needed but I will look for the pocket one. Thanks for the catch!